catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/forms/FilterNodeAssertionSupport.java

```diff
@@ -65,16 +80,16 @@ public static void assertTemplatedNode(
       String expectedProperty,
       String defaultValue,
       String nodeId) {
-    assertThat(node.getOperator(), is(expectedType));
-    assertThat(node.isLeaf(), is(true));
+    assertThat(node, is(instanceOf(FunctionFilterNode.class)));
+    FunctionFilterNode functionFilterNode = (FunctionFilterNode) node;
+
+    assertThat(functionFilterNode.getOperator(), is(expectedType));
 
-    assertThat(node.getProperty(), is(expectedProperty));
-    assertThat(node.getValue(), is(nullValue()));
-    assertThat(node.isTemplated(), is(true));
+    assertThat(functionFilterNode.getValue(), is(nullValue()));
 
-    Map<String, Object> templateProps = node.getTemplateProperties();
-    assertThat(templateProps.get("defaultValue"), is(defaultValue));
-    assertThat(templateProps.get("nodeId"), is(nodeId));
+    Map<String, Object> templateProps = functionFilterNode.getFunctionArguments();
+    assertThat(((List<Object>) templateProps.get("params")).get(0), is(defaultValue));
```

Comment:
I wonder if `assertThat(templateProps, hasEntry(is("params"), contains(defaultValue, nodeId)))` would work.

Reply:
![screen shot 2018-07-06 at 7 03 58 am](https://user-images.githubusercontent.com/17837152/42383001-cc975bea-80ea-11e8-8f93-9f532f310de1.png)

Reply:
Ok, I think this will work instead: `assertThat(templateProps, hasEntry("params", asList(defaultValue, nodeId)))`